ClothesApp/Application/Repositories/IBrandsRepository.cs

```diff
@@ -0,0 +1,12 @@
+using Domain.Entities;
+
+namespace Application.Repositories;
+
+public interface IBrandsRepository : IBaseRepository<Brand>
+{
+    Task<bool> DoesBrandExist(string name);
+
+    Task<bool> DoesBrandExist(long id);
```

Comment:
I think this method is also very generic, if you have get by id, then exists by id is also common